packages/services/document-service/src/main/java/software/uncharted/terarium/documentserver/models/xdd/Document.java

```diff
@@ -16,13 +16,13 @@
 @Accessors(chain = true)
 public class Document implements Serializable {
 
-	@JsonbProperty("_gddid")
+	@JsonbProperty("gddid")
```

Comment:
Right now in this PR you won't actually see the results of changing this, as the `Document` object is never used in the serialization (currently this is being addressed in #447 )  However, we can't get rid of `"_gddid" as this is the field name coming to us from xDD, and changing this to not have the underscore results in it not being serialized properly.

Reply:
Okay I will revert this change. I believe I did something similar for `abstract`/`abstracttext` 
To be clear the remapping of this fieldname should be done on the front end then?